.changeset/late-files-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kadena/tools": patch
+"@kadena/react-ui": patch
+---
+
+Refactor Accordion component
```

Comment:
In this case I wouldn't add a changeset to @kadena/tools as (i think) there is not something substantial that changed in the tools site. For @kadena/react-ui I would also update it to `none` since it isn't published, but we can also have a discussion about it if you disagree.

Also would make the description of the change more descriptive as it is meant to help users understand what has changed